forge/routes/api/team.js

```diff
@@ -616,6 +616,43 @@ module.exports = async function (app) {
                     reply.send(app.db.views.Team.team(request.team))
                     return
                 }
+            } else if (Object.hasOwn(request.body, 'suspended')) {
+                if (Object.keys(request.body).length > 1) {
+                    reply.code(400).send({ code: 'invalid_request', error: 'Cannot modify other properties whilst changing suspended state' })
+                    return
+                }
+                if (!!request.body.suspended !== !!request.team.suspended) {
```

Comment:
Is there a diliberate reason the schema is not update to include `suspended`?

Reply:
The schema was already incomplete as its missing the 'type' option that can be passed in. I'll tidy it up.